### ISSUE TITLE
Support React 0.13

### ISSUE THREAD

**Issue by pilwon** [react-bootstrap/react-bootstrap#420]
```
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "npm" "install" "react-bootstrap"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-bootstrap@0.15.1 wants react@>=0.12

npm ERR! Please include the following file with any support request:
npm ERR!     npm-debug.log
```

[React 0.13 has already reached RC1](http://facebook.github.io/react/blog/2015/02/24/react-v0.13-rc1.html).
[The popular react-router library is also working on 0.13 support](https://github.com/rackt/react-router/milestones/React%200.13).
It'd be great if `react-bootstrap` officially supports React 0.13 as well.

**Comment by steos** [comment]
Hey. React 0.13.0-rc2 works fine for me with latest react-bootstrap (0.16.1). What's blocking this issue? I need React 0.13 for seamless ImmutableJS support.

**Comment by meric** [comment]
The npm package could use an update to the peerDependencies row.

**Comment by mtscout6** [comment]
We need to ensure we don't break the docs.

**Comment by bastiao** [comment]
+1 for update.

**Comment by pedroteixeira** [comment]
It seems to be working fine with 0.13

**Comment by wmertens** [comment]
@steos seamless immutablejs support? Tell me more!

**Comment by steos** [comment]
@wmertens well depends on how you define seamless I guess... As of 0.13 React supports iterators and immutable-js sequences as children.

**Comment by mtscout6** [comment]
Resolved by #424
